src/linear_algebra/trace.lean

```diff
@@ -115,6 +116,28 @@ end
 section
 variables (R : Type u) [field R] {M : Type v} [add_comm_group M] [module R M]
 
+/-- The trace of a linear map correspond to the contraction pairing under the isomorphism
+ `M →ₗ M ≃ M* ⊗ M`-/
```

Comment:
This is a mix of informal and formal notation and could also use some brackets. I think it's fine to use either but not a mix. For example if you opt for a formal notation then this should be:
```suggestion
 `(M →ₗ[R] M) ≃ (module.dual R M) ⊗ M`-/
```
or even `module.End R M ≃ (module.dual R M) ⊗ M`

Reply:
Changed to informal notation only